server/db/migrations/20200131013131-create-user.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
```

Comment:
Also, what's the purpose of server/db/migrations?

Reply:
We use migrations to keep track of changes to the database. Maybe we shouldn't push this migration/seed files into this public repository?

Reply:
Is there a reason we should publish these? This means everyone on github can see them.

Reply:
No. I think it was a mistake. I will delete these files immediately.